spec/std/yaml/any_spec.cr

```diff
@@ -7,6 +7,14 @@ describe YAML::Any do
       YAML.parse("").as_nil.should be_nil
     end
 
+    it "gets bool" do
+      YAML.parse("true").as_bool.should be_true
+      YAML.parse("false").as_bool.should be_false
+      YAML.parse("true").as_bool?.should be_true
+      YAML.parse("false").as_bool?.should be_false
+      YAML.parse("2").as_bool?.should be_nil
+    end
+
```

Comment:
Did you know that "yes" and "no" are considered booleans in YAML? http://yaml.org/type/bool.html

Reply:
Surprising :open_mouth: 
If we stick with the YAML 1.2 spec, this isn't the case any more http://yaml.org/spec/1.2/spec.html#id2803629

Reply:
Yeah,

> In this version of YAML, boolean values may be serialized as “true” or “false”; the empty scalar as “null”.

Totally makes sense, TIL!